tests/integration/kubernetes/gha-run.sh

```diff
@@ -482,7 +498,17 @@ function deploy_nydus_snapshotter() {
 	popd
 
 	kubectl rollout status daemonset nydus-snapshotter -n nydus-system --timeout ${SNAPSHOTTER_DEPLOY_WAIT_TIMEOUT}
-
+	if [ "${PULL_TYPE}" == "host-share-image-block" ]; then
```

Comment:
Likewise move this change to https://github.com/kata-containers/kata-containers/pull/7837

Reply:
Done.